cmake/projects/LAPACK/hunter-msvc.cmake

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2017, NeroBurner
+# All rights reserved.
+
+#hunter_cacheable(LAPACK) # install is always cacheable
+
+# determine architecture
+include(hunter_check_toolchain_definition)
+hunter_check_toolchain_definition(NAME "_WIN64"    DEFINED WIN64_defined)
+
+if(WIN64_defined)
+  hunter_add_version(
+    PACKAGE_NAME
+    LAPACK
+    VERSION
+    "3.7.1"
+    URL
+    https://github.com/NeroBurner/lapack-prebuilt/archive/v3.7.1-msvc-amd64.zip
```

Comment:
I've saved it to hunter-packages/lapack, please use this link:
* https://github.com/hunter-packages/lapack/releases/download/v3.7.1-p0/lapack-prebuilt-3.7.1-msvc-amd64.zip